src/components/loading/Loading.tsx

```diff
@@ -0,0 +1,40 @@
+import { IonCol, IonGrid, IonRow, IonText } from '@ionic/react'
+import React from 'react'
+
+import KwizAnimated from '../animations/KwizAnimated'
+
+const Loading = () => {
+  return (
+    <div
+      style={{
+        height: '100%',
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        background:
+          'linear-gradient(0deg, rgba(173,220,244,1) 0%, rgba(235,250,248,1) 100%)',
+      }}
+    >
+      <IonGrid>
+        <IonRow>
+          <IonGrid>
+            <IonRow>
+              <IonCol>
+                <KwizAnimated />
```

Comment:
What is this for? There is no children here that would be animated? :thinking:

Reply:
What do yo mean? The `KwizAnimated` component is not a wrapper that you can out around other components. It's a component which shows the kwiz-logo and animates it using the keyframes that I have specified.

Reply:
Aah. I was confused by the diff just above this and thought it was KwizAnimated. Instead it is `BlurAnimated`, which has a children prop.